includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -739,11 +788,24 @@ public function payment_confirm() {
 	public function return_handler() {
 		$raw_body = file_get_contents( 'php://input' );
 
-		$this->log( sprintf( 'IPN: Initialized %s from %s', $_SERVER['REQUEST_URI'], $_SERVER['REMOTE_ADDR'] ) );
-		$this->log( sprintf( 'Incoming Callback. Post data: %s', var_export( $raw_body, true ) ) );
+		$this->adapter->log(
+			LogLevel::INFO,
+			sprintf(
+				'Incoming Callback: Initialized %s from %s',
+				$_SERVER['REQUEST_URI'],
+				$_SERVER['REMOTE_ADDR']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized input variable: $_SERVER['REMOTE_ADDR']](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-checkout/pullRequest?prid=5510424)